Caw.cc

```diff
@@ -0,0 +1,63 @@
+#include <iostream>
+#include <string>
+
+#include "Caw.h"
```

Comment:
Usually the .h for a .cc file goes at the top

Reply:
Reformatted (through clang)